Adding Codebyte for getUTCHours (#4161)
Codecademy/docs: content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md

```shell
2
```

> ### Codebyte Example

Reviewer: ```suggestion
## Codebyte Example
```
It should be a H2 subheading.

> In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hour of the dates according to universal time. Run this example in the Codebyte to test the results. 

Reviewer: Give space before and after the code byte.

> In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hour of the dates according to universal time. Run this example in the Codebyte to test the results. 

Reviewer: ```suggestion
In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hours of the dates according to universal time. You can just run this example in the Codebyte to test the results. 
```

```shell
2
```

## Codebyte Example

> In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hours of the dates according to universal time. You can just run this example in the Codebyte to test the results. 

Reviewer: ```suggestion
In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hours of the dates according to universal time. You can just run this example in the Codebyte to test the results. 

```
